build_che.sh

```diff
@@ -14,11 +14,4 @@ if [ $? -ne 0 ]; then
   echo "Error building che/rh-che with dashboard"
   exit 1;
 fi
-cp target/export/che-dependencies/che/assembly/assembly-main/target/eclipse-che-*.tar.gz ${HomeDir}
```

Comment:
Or simply don't copy the distribution we don't want to push to docker by commenting the following lines:
https://github.com/kbsingh/build-run-che/blob/master/build_che.sh#L17
and 
https://github.com/kbsingh/build-run-che/blob/master/build_che.sh#L24
This should be sufficient no ?